src/ChronosInterval.php

```diff
@@ -256,22 +266,16 @@ public function __get(string $name)
             case 'seconds':
                 return $this->s;
 
+            case 'microseconds':
+                return (int)($this->f * 1000000);
+
             case 'weeks':
                 return (int)floor($this->d / ChronosInterface::DAYS_PER_WEEK);
 
             case 'daysExcludeWeeks':
             case 'dayzExcludeWeeks':
                 return $this->dayz % ChronosInterface::DAYS_PER_WEEK;
-            case 'days':
-                return $this->days;
-            case 'y':
-            case 'm':
-            case 'd':
-            case 'h':
-            case 'i':
-            case 's':
-            case 'invert':
-                return parent::__get($name);
```

Comment:
Why is this being removed?

Reply:
These public properties exist on DateInterval.  This isn't required for the PR to work, but it confused me when investigating where to add ->f and ->microseconds support.